client/components/common/ContentPreview/Element.vue

```diff
@@ -40,17 +62,32 @@ export default {
 .element-preview-container {
   display: flex;
   position: relative;
+  margin: 1rem 0;
+  float: none;
```

Comment:
Along with this one, there is also an `elementWidth` class binding which sets the `float: left` to this element. I've put it here to annul that property (since it results in 0 height parent).